mantle/kola/tests/rhcos/luks.go

```diff
@@ -35,6 +38,84 @@ func init() {
 			}
 		}`),
 	})
+	register.RegisterTest(&register.Test{
+		Run:                  luksTangTest,
+		ClusterSize:          0,
```

Comment:
nit: A `ClusterSize` of 0 looks odd. Consider adding a link to [the conversation](https://github.com/coreos/coreos-assembler/pull/1310#discussion_r401908836) as context.

Reply:
Will do.